### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Still said Droid-ify for the translations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,17 +23,17 @@
...
 |:---------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------:|
 
-### :last_quarter_moon_with_face: Dark theme
+### :last_quarter_moon_with_face: Dark
 
 | <img src="metadata/en-US/images/phoneScreenshots/home-dark.png" width="500" align="center"/> | <img src="metadata/en-US/images/phoneScreenshots/app-dark.png" width="500" align="center"/> |
 |:--------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------:|
 
-### :waxing_crescent_moon: Amoled theme
+### :waxing_crescent_moon: Black
```
Comment (Iamlooker) [inline README.md:36]: Black theme*

File: README.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 
-[![Github repo stars](https://img.shields.io/github/stars/Iamlooker/Droid-ify?style=flat)](https://github.com/Iamlooker/Droid-ify/stargazers)
-[![Github License](https://img.shields.io/github/license/Iamlooker/Droid-ify)](https://github.com/Iamlooker/Droid-ify/blob/master/COPYING)
-[![Github All Releases](https://img.shields.io/github/downloads/Iamlooker/Droid-ify/total.svg)](https://github.com/Iamlooker/Droid-ify/releases/)
-[![Github release](https://img.shields.io/github/v/release/Iamlooker/Droid-ify?display_name=tag)](https://github.com/Iamlooker/Droid-ify/releases/latest)
+[![GitHub repo stars](https://img.shields.io/github/stars/Iamlooker/Droid-ify?style=flat)](https://github.com/Iamlooker/Droid-ify/stargazers)
+[![GitHub License](https://img.shields.io/github/license/Iamlooker/Droid-ify)](https://github.com/Iamlooker/Droid-ify/blob/master/COPYING)
+[![GitHub All Releases](https://img.shields.io/github/downloads/Iamlooker/Droid-ify/total.svg)](https://github.com/Iamlooker/Droid-ify/releases/)
+[![GitHub release](https://img.shields.io/github/v/release/Iamlooker/Droid-ify?display_name=tag)](https://github.com/Iamlooker/Droid-ify/releases/latest)
```
Comment (machiav3lli) [inline README.md:8]: I guess links can also be updated

File: README.md (hunk 1 above)
Comment (machiav3lli) [reply README.md:36]: I agree, this applies to the three themes

Comment (comradekingu) [reply README.md:36]: I think it goes without saying, but OK.

**Pull request merged**